airbyte-server/src/main/java/io/airbyte/server/ConfigDumpImporter.java

```diff
@@ -119,18 +127,17 @@ public ImportRead importData(File archive) {
 
         // 2. dry run
         try {
-          checkImport(sourceRoot);
+          checkImport(targetVersion, sourceRoot, seedPath);
         } catch (Exception e) {
-          LOGGER.warn("Dry run failed, setting DB version back to initial version");
-          postgresPersistence.setVersion(initialVersion);
```

Comment:
@subodh1810 is this line necessary? Why do we need to reset the version here? I can't find anywhere before this line where this class would have changed it. Lmk if I'm missing anything. If it's not needed then we can make it so this class doesn't need to know about `initialVersion` at all.

Reply:
Yeah you are correct, this line is not necessary. We dont change the DB version before this call. I changed it as part of my PR https://github.com/airbytehq/airbyte/pull/4626 but forgot to remove this